rpcs3/Emu/RSX/Overlays/overlay_save_dialog.cpp

```diff
@@ -104,24 +105,59 @@ namespace rsx
 			m_description->back_color.a    = 0.f;
 			m_time_thingy->back_color.a    = 0.f;
 
+			fade_animation.duration = 0.15f;
+
 			return_code = selection_code::canceled;
 		}
 
+		void save_dialog::close(bool use_callback, bool stop_pad_interception)
+		{
+			fade_animation.current = color4f(1.f);
+			fade_animation.end = color4f(0.f);
+			fade_animation.active = true;
+
+			close_requested = get_system_time() + 150000;
+			close_use_callback = use_callback;
+			close_stop_pad_interception = stop_pad_interception;
+		}
+
 		void save_dialog::update()
 		{
 			m_time_thingy->set_text(date_time::current_time());
 			m_time_thingy->auto_resize();
+
+			const u64 t = get_system_time();
+			if (!creation_time)
+			{
+				// First tick
+				creation_time = t;
+				Emu.GetCallbacks().play_sound(fs::get_config_dir() + "sounds/snd_system_ok.wav");
+				return;
+			}
+
+			if (fade_animation.active)
+			{
+				fade_animation.update(rsx::get_current_renderer()->vblank_count);
+			}
+
+			if (close_requested && t >= close_requested)
+			{
+				fade_animation.active = false;
```

Comment:
I think this is placebo and actually wrong.
you'll re-enable input if you do this.